dvc/remote/base.py

```diff
@@ -142,6 +138,22 @@ def _check_requires(self, config):
         ).format(url, missing, " ".join(missing), self.scheme)
         raise RemoteMissingDepsError(msg)
 
+    def _get_checksum_jobs(self, config):
+        checksum_jobs = config.get(Config.SECTION_REMOTE_CHECKSUM_JOBS)
+        if checksum_jobs:
+            return checksum_jobs
+
+        if self.repo:
+            try:
+                core = self.repo.config.config.get(Config.SECTION_CORE, {})
+                return core.get(
+                    Config.SECTION_CORE_CHECKSUM_JOBS, self.CHECKSUM_JOBS
+                )
+            except ConfigError:
```

Comment:
Hm, what would raise ConfigError though? Looks like you are accessing the dics directly.